Automattic/wp-calypso
# Nav redesign: make tab content layout consistent

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/6899
- https://github.com/Automattic/dotcom-forge/issues/6828

## Proposed Changes

This PR applies layout fixes as suggested by the above issue:

- Make the tabs left-aligned with the site icon
- Make tab content area left-aligned with the "Overview" tab  
- Set max-width of the tab content area to 1400px.

| Before | After|
|-|-|
|<img width="909" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/7572f4a7-fe45-4233-b588-a85bfe70b166">|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/f5d9ffe5-ad26-4e43-aec5-d71acecd036c">|
|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/33c4124d-a60f-4444-b631-22da5e0157f0">|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/6fd0fbed-08ee-4e46-8868-df48a5b57a8a">|
|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/db2d4204-9ea6-4658-9c84-41ff88c72031">|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/f0609b0b-5ea4-46d0-be9c-495ca1b15443">|
|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/fbfbd83a-039e-429e-aa29-42a3046d146c">|<img width="910" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/c63451a7-90d8-484e-9581-58f657b02124">|


## Testing Instructions

1. Go to `<Calypso Live URL>/sites`
2. Go to all tabs and verify that they have consistent layout as described above.
3. Also test with various screen width and make sure they don't break.
4. Go to `/hosting-config/:site` and `/site-monitoring/:site` with `flags=-layout/dotcom-nav-redesign-v2` and verify that there are no regressions found on old pages (when the flag is turned off).

## Pre-merge Checklist
